app/javascript/advertisements/media/theme.scss

```diff
@@ -0,0 +1,132 @@
+#cf[data-template='media'] {
+  &[data-theme='dark'] {
+    $text-color: rgba(255, 255, 255, 0.8);
+    $bold-text-color: #ffffff;
+    $link-color: #ffffff;
+    $powered-by-color: rgba(255, 255, 255, 0.5);
+
+    .cf-text {
+      color: $text-color;
+      strong {
+        color: $bold-text-color
+      }
+    }
+    .cf-link { color: $link-color; }
+    .cf-powered-by { color: $powered-by-color; }
+  }
+
+  &[data-theme='light'] {
+    $text-color: rgba(0, 0, 0, 0.8);
+    $bold-text-color: rgba(0, 0, 0, 0.8);
+    $link-color: #2c6ac7;
+    $powered-by-color: rgba(0, 0, 0, 0.5);
+
+    .cf-text {
+      color: $text-color;
+      strong {
+        color: $bold-text-color
+      }
+    }
+    .cf-link { color: $link-color; }
+    .cf-powered-by { color: $powered-by-color; }
+  }
+  
+  margin: 0 auto;
+  max-width: 330px;
+
+  .cf-wrapper {
+    display: flex;
+    font-size: 13px;
+    line-height: 1.3;
+    text-align: left;
+  }
+
+  .cf-img-wrapper {
+    flex: 0 0 60px;
+  }
+
+  .cf-img-wrapper img {
+    border-radius: 5px;
+    position: relative;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  .cf-text {
+    font-weight: 350;
+    letter-spacing: 0.5px;
+  }
+
+  .cf-text strong {
+    font-weight: bold;
+  }
+
+  .cf-link {
+    text-decoration: underline;
+    font-weight: 400;
+  }
+
+  .cf-powered-by {
+    display: block;
+    font-size: 11px;
+    margin-top: 5px;
+  }
+}
+
+// #cf[data-template='media'][data-theme='light'] {
```

Comment:
Any reason this is all commented out?

Reply:
Look like @coderberry refactored this stylesheet? My guess is that this could be deleted now?

Reply:
Nope. This should have remained. Right about this point is when my wife said "You need to be done with work now". You'll likely need to uncomment and verify it looks good.